### PR TITLE
Complete docstrings for reactionMenus, lib, event categories, menus, and pingables.

### PR DESCRIPTION
**Pull request by Trimatix**

### CODE REVIEW

File: src/esportsbot/bot.py [hunk 1]
```diff
@@ -13,14 +14,26 @@
...
+# Value to assign new guilds in their role_ping_cooldown_seconds attribute
 DEFAULT_ROLE_PING_COOLDOWN = timedelta(hours=5)
+# Value to assign new guilds in their pingme_create_poll_length_seconds attribute
 DEFAULT_PINGME_CREATE_POLL_LENGTH = timedelta(hours=1)
+# Value to assign new guilds in their pingme_create_threshold attribute
 DEFAULT_PINGME_CREATE_THRESHOLD = 6
+
+# EsportsBot client instance
 client = lib.client.instance()
+# TODO
```
Comment (Laura7089) [inline src/esportsbot/bot.py:26]: :eyes:

Comment (Laura7089) [inline src/esportsbot/bot.py:26]: :eyes:

File: src/esportsbot/bot.py [hunk 2]
```diff
@@ -34,6 +47,7 @@ async def send_to_log_channel(guild_id, msg):
 
 @client.event
 async def on_ready():
+    # Initialize the reactionMenuDB and pingme role cooldowns, since this can't be done synchronously
```
Comment (Laura7089) [inline src/esportsbot/bot.py:50]: Do these one-line comments work as docstrings?

File: src/esportsbot/reactionMenus/reactionMenu.py [hunk 3]
```diff
@@ -427,9 +439,11 @@ def getMenuEmbed(self) -> Embed:
         return menuEmbed
 
 
-    async def updateMessage(self, noRefreshOptions=False):
+    async def updateMessage(self, noRefreshOptions : bool = False):
```
Comment (Laura7089) [inline src/esportsbot/reactionMenus/reactionMenu.py:442]:
```suggestion
    async def updateMessage(self, noRefreshOptions : bool = False):
    async def updateMessage(self, noRefreshOptions: bool = False):
```

File: src/esportsbot/bot.py (hunk 1 above)
Comment (Trimatix) [reply src/esportsbot/bot.py:26]: new issue made: https://github.com/FragSoc/esports-bot/issues/78

File: src/esportsbot/bot.py (hunk 2 above)
Comment (Trimatix) [reply src/esportsbot/bot.py:50]: You're right they don't, let me fix that..

**Review by Laura7089 — APPROVED** [review]

**Pull request merged**